eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -324,4 +324,56 @@
   <!-- Don't enforce non-static ClientLogger instances in azure-core-mgmt PollerFactory and PollingState types-->
   <suppress checks="com\.azure\.tools\.checkstyle\.checks\.(ThrowFromClientLoggerCheck|GoodLoggingCheck)" files="com.azure.core.management.implementation.polling.PollerFactory.java"/>
   <suppress checks="com\.azure\.tools\.checkstyle\.checks\.(ThrowFromClientLoggerCheck|GoodLoggingCheck)" files="com.azure.core.management.implementation.polling.PollingState.java"/>
+  
+  <!-- Cosmos sdk suppressions -->
+  <suppress checks="[a-zA-Z0-9]*" files="[/\\]azure-cosmos[/\\]src[/\\]test[/\\]"/>
+  <suppress checks="[a-zA-Z0-9]*" files="[/\\]cosmos[/\\]examples[/\\]"/>
+  <suppress checks="[a-zA-Z0-9]*" files="[/\\]cosmos[/\\]implementation[/\\]"/>
+  <suppress checks="[a-zA-Z0-9]*" files="com.azure.cosmos.BridgeInternal.java"/>
+  <suppress checks="[a-zA-Z0-9]*" files="com.azure.cosmos.CosmosBridgeInternal.java"/>
+  <suppress checks="[a-zA-Z0-9]*" files="com.azure.cosmos.CommonsBridgeInternal.java"/>
+  <suppress checks="[a-zA-Z0-9]*" files="[/\\]cosmos[/\\]rx[/\\]examples[/\\]"/>
+  <suppress checks="[a-zA-Z0-9]*" files="[/\\]azure-cosmos-benchmark[/\\]"/>
+  <suppress checks="com\.azure\.tools\.checkstyle\.checks\.GoodLoggingCheck"
+            files="com.azure.cosmos.CosmosResponseDiagnostics"/>  
+  <suppress checks="com\.azure\.tools\.checkstyle\.checks\.ThrowFromClientLoggerCheck"
```

Comment:
Is possible to group `ThrowFromClientLoggerCheck` suppression into one together. 
You can follow what search suppression:
```
  <suppress checks="." files=".*[/\\]search[/\\]implementation[/\\]((?!(GeoPointDeserializer|Iso8601DateDeserializer|SerializationUtil)).*)"/>
```
 `GeoPointDeserializer`, `Iso8601DateDeserializer`, `SerializationUtil` are sharing same package path
`com.azure.search.implementation`.

Reply:
Thanks for the suggestion, grouped similar ones together.